src/Game.cpp

```diff
@@ -351,6 +352,18 @@ void Game_setLocalSpeed(Game* self, int speed) {
     reinterpret_cast<BWAPI::Game*>(self)->setLocalSpeed(speed);
 }
 
+bool Game_issueCommand(Game* self, UnitIterator* units, UnitCommand command) {
+    IteratorBase* iter = reinterpret_cast<IteratorBase*>(units);
+    assert(iter->id() == itUnit);
+    BWAPI::Unitset unitset;
+    for (; iter->valid(); iter->next()) {
+        Unit* const unit = reinterpret_cast<Unit*>( iter->get() );
```

Comment:
inconsistent style, I'd propose to remove the spacing:
`reinterpret_cast<Unit*>(iter->get());`